### PR TITLE
SPORTS-239 Add knockouts routes

### PR DESCRIPTION
**Pull request by sanjeevkpandit**
- Show a tree structure of the knockouts

image

### CODE REVIEW

File: web-app/src/views/common/knockouts/Knockouts.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      (lastRoundFixtures[0].homeTeamParentFixtureId || lastRoundFixtures[0].awayTeamParentFixtureId)
+    ) {
+      return true;
+    }
+
+    return false;
+  }
+
+  get latestRoundFixtureIds() {
+    const ids = this.groupedFixtures[0].fixtures.map((a) => a.id);
```
Comment (SAGARACH65) [inline web-app/src/views/common/knockouts/Knockouts.vue]:
```suggestion
    const ids = this.groupedFixtures[0].fixtures.map(a => a.id);
```

File: web-app/src/views/common/knockouts/Knockouts.vue [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    const lastRoundFixtures = this.groupedFixtures && this.groupedFixtures.length && this.groupedFixtures[0].fixtures;
+
+    if (lastRoundFixtures && lastRoundFixtures.length > 1) {
+      return true;
+    }
+
+    if (
+      lastRoundFixtures &&
+      lastRoundFixtures.length === 1 &&
```
Comment (SAGARACH65) [inline web-app/src/views/common/knockouts/Knockouts.vue]: Let's refactor this. We are checking the same condition and returning `true` above this as well.

File: web-app/src/assets/sass/components/_manifest.scss [hunk 3]
```diff
@@ -25,3 +26,5 @@
 @import 'modal';
 @import 'filters';
 @import 'registration';
+@import 'knockout';
```
Comment (SAGARACH65) [inline web-app/src/assets/sass/components/_manifest.scss]: Is the order necessary here? If not let's maintain a ladder here as well

File: web-app/src/views/common/knockouts/Knockouts.vue (hunk 1 above)
Comment (SAGARACH65) [inline web-app/src/views/common/knockouts/Knockouts.vue]: Shouldn't we check for the condition where  `this.groupedFixtures` is an empty array

File: web-app/src/assets/sass/components/_manifest.scss (hunk 3 above)
Comment (sanjeevkpandit) [reply web-app/src/assets/sass/components/_manifest.scss]: Yes, the order matters.

**Pull request merged**